Systematics/test/workspaceStd.py

```diff
@@ -670,7 +769,66 @@
+        print "Signal MC central value, so dumping PDF weights"
+        dumpPdfWeights = True
+        nPdfWeights = 60
+        nAlphaSWeights = 2
+        nScaleWeights = 9
+    else:
+        print "Data, background MC, or non-central value, or no systematics: no PDF weights"
+        dumpPdfWeights = False
+        nPdfWeights = -1
+        nAlphaSWeights = -1
+        nScaleWeights = -1
+    if not customize.processId == "Data":
+        mH = None
+        ldset=""
+        if customize.datasetName():
+            ldset = customize.datasetName().lower() 
+        if "htogg" in ldset or "tthjettogg" in ldset:
+            try:
+                mH = float(customize.datasetName().split("_M")[1].split("_")[0])
+            except Exception, e:
+                print(e,customize.datasetName())
+                pass
+        NNLOPSreweight=False
+        genToReweight=None
+        if ( customize.datasetName() and customize.datasetName().count("GluGlu") ):
+            print "datasetName contains GluGlu --> NNLOPSrewwight is True"
+            NNLOPSreweight=True
+            if customize.datasetName().count("amcatnlo"):
+            #                print "datasetName contains amcatnlo --> gen to be reweighted is amcatnlo"
+                genToReweight = "amcatnlo"
+            if customize.datasetName().count("powheg"):
+            #                print "datasetName contains powheg --> gen to be reweighted is powheg"
+                genToReweight = "powheg"
+        print 'pdfWeights in worspaceStd'
+        fc.addGenOnlyAnalysis(process,customize.processId,process.flashggTagSequence,
+                              customize.acceptance,tagList,systlabels,NNLOPSreweight,genToReweight,
+                              pdfWeights=(dumpPdfWeights,nPdfWeights,nAlphaSWeights,nScaleWeights),
+                              mH=mH,filterEvents=customize.filterNonAcceptedEvents)
+        pdfWeights=(dumpPdfWeights,nPdfWeights,nAlphaSWeights,nScaleWeights),
+        print pdfWeights
+
+# if customize.tthTagsOnly:
+#     process.p = cms.Path(process.dataRequirements*
+#                          process.genFilter*
+#                          process.flashggDiPhotons* # needed for 0th vertex from microAOD
+#                          process.flashggUpdatedIdMVADiPhotons*
+#                          process.flashggDiPhotonSystematics*
+#                          process.flashggMetSystematics*
+#                          process.flashggMuonSystematics*process.flashggElectronSystematics*
+#                          (process.flashggUnpackedJets*process.jetSystematicsSequence)*
+#                          (process.flashggTagSequence*process.systematicsTagSequences)*
+#                          process.flashggSystTagMerger*
+#                          process.penultimateFilter*
+#                          process.finalFilter*
+#                          process.tagsDumper)
+if not customize.processId=="Data" :
```

Comment:
after some testing it looks like the logic has been changed here - presumably unintended? This file also needs cleanup in general, so please have a look (did you mean to replace with fiducialStd?)

Reply:
In principle everything for the fiducial went from workspaceStd to fiducialStd. I will remove everything from here. For workspaceStd, the idea was to take whatever is there in the master. Is missed this bc there was no conflict here. Sorry for that.